sapphire/analysis/reconstructions.py

```diff
@@ -447,3 +441,17 @@ def _store_reconstruction(self, coincidence, core_x, core_y, theta, phi,
             row['s%d' % number] = True
 
         row.append()
+
+    def _get_active_stations(self):
+        """Return station numbers with non-empty event table in datafile"""
+        def stations_with_events():
```

Comment:
The generator is more readable, faster and shorter than the list.
But I'll change it.